### ISSUE TITLE
There is a idea or bug what i want to say

### ISSUE THREAD

**Issue by aishuchen** [crsmithdev/arrow#736]
## Feature Request

I'm sorry my English is poor.
When I used `arrow.get(timestamp, tzinfo=tzinfo)` to get a arrow object,  the `get` method didn't use the param `tzinfo`. So I have to use `arrow.get(timestamp).to('Asia/Shanghai')` and get the arrow object what I needed. Is it a bug?

**Comment by systemcatch** [comment]
Hello @aishuchen, in #719 I implemented the feature you're looking for on the master branch.

```shell
(arrow) chris@ThinkPad:~/arrow$ python
Python 3.7.4 (default, Sep 19 2019, 11:01:37) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import arrow
>>> arrow.__version__
'0.15.4'
>>> from dateutil import tz
>>> import time
>>> timezone = tz.gettz("Asia/Shanghai")
>>> timestamp = time.time()
>>> arrow.get(timestamp, tzinfo=timezone)
<Arrow [2019-12-12T20:03:23.552349+08:00]>
```

This feature will be available in the next release of arrow, which will likely happen before Christmas.

**Comment by aishuchen** [comment]
> Hello @aishuchen, in #719 I implemented the feature you're looking for on the master branch.
> 
> ```shell
> (arrow) chris@ThinkPad:~/arrow$ python
> Python 3.7.4 (default, Sep 19 2019, 11:01:37) 
> [GCC 5.4.0 20160609] on linux
> Type "help", "copyright", "credits" or "license" for more information.
> >>> import arrow
> >>> arrow.__version__
> '0.15.4'
> >>> from dateutil import tz
> >>> import time
> >>> timezone = tz.gettz("Asia/Shanghai")
> >>> timestamp = time.time()
> >>> arrow.get(timestamp, tzinfo=timezone)
> <Arrow [2019-12-12T20:03:23.552349+08:00]>
> ```
> 
> This feature will be available in the next release of arrow, which will likely happen before Christmas.

Got it! Thanks.

**Comment by aishuchen** [comment]
I upgraded the `arrow` package, and I found the bug has not been fixed. ps: specia `timestamp` 1575043200

```
Python 3.7.5 (v3.7.5:5c02a39a0b, Oct 14 2019, 18:49:57) 
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import arrow
>>> from dateutil import tz
>>> timezone = tz.gettz("Asia/Shanghai")
>>> timestamp = 1575043200
>>> arrow.get(timestamp)
<Arrow [2019-11-29T16:00:00+00:00]>
>>> arrow.get(timestamp, tzinfo=timezone)
<Arrow [2019-11-29T16:00:00+00:00]>
>>> arrow.get(timestamp).to(timezone)
<Arrow [2019-11-30T00:00:00+08:00]>
>>> arrow.__version__
'0.15.4'
>>> 
```

**Comment by jadchaar** [comment]
Hey @aishuchen, this fix should come with Arrow v0.15.5. We are nailing down a small bug with our test suite before releasing it, but stay tuned! It should be shipped soon.

**Comment by jadchaar** [comment]
Hey @aishuchen, v0.15.5 is now released!
